resources/views/waitingrooms/show.blade.php

```diff
@@ -10,8 +10,7 @@
     <div class="container">
         <div class="wrapper">
             <div class="container__block">
-                <p>Campaign ID: {{ $room->campaign_id }}</p>
```

Comment:
i know these aren't on the room, but it might be helpful to show it from the contest